File: stripe/src/androidTest/java/com/stripe/android/AddPaymentMethodActivityTest.java
package com.stripe.android;

import android.support.test.runner.AndroidJUnit4;

import com.stripe.android.view.AddPaymentMethodActivity;

import org.junit.Rule;
import org.junit.Test;
import org.junit.runner.RunWith;

import static android.support.test.espresso.Espresso.onView;
import static android.support.test.espresso.assertion.ViewAssertions.matches;
import static android.support.test.espresso.matcher.ViewMatchers.isDisplayed;
import static android.support.test.espresso.matcher.ViewMatchers.withText;

@RunWith(AndroidJUnit4.class)
public class AddPaymentMethodActivityTest {

    @Rule
    public ActivityTestRule<AddPaymentMethodActivity> mActivityRule =
            new ActivityTestRule(AddPaymentMethodActivity.class);

    @Test
    public void titleRenders() {
        onView(withText(R.string.title_add_a_card)).check(matches(isDisplayed()));
    }

}

File: stripe/src/androidTest/java/com/stripe/android/AddPaymentMethodActivityTest.kt
package com.stripe.android

import androidx.test.espresso.Espresso.onView
import androidx.test.espresso.assertion.ViewAssertions.matches
import androidx.test.espresso.matcher.ViewMatchers.isDisplayed
import androidx.test.espresso.matcher.ViewMatchers.withText
import androidx.test.ext.junit.runners.AndroidJUnit4
import androidx.test.rule.ActivityTestRule
import androidx.test.runner.AndroidJUnit4
import com.stripe.android.view.AddPaymentMethodActivity
import kotlin.test.Test
import org.junit.Rule
import org.junit.runner.RunWith

@RunWith(AndroidJUnit4::class)
class AddPaymentMethodActivityTest {

    @Rule
    var mActivityRule = ActivityTestRule(AddPaymentMethodActivity::class.java)

    @Test
    fun titleRenders() {
        onView(withText(R.string.title_add_a_card)).check(matches(isDisplayed()))
    }
}
